oneview-ansible.md

```diff
@@ -12543,15 +12659,17 @@ Retrieve the facts about one or more of the OneView Users.
  Retrieve the facts about one or more of the Users from OneView.
 
 #### Requirements (on the host that executes the module)
-  * python >= 2.7.9
+  * python >= 3.4.2
   * hpeOneView >= 5.4.0
```

Comment:
change hpeOneView version to latest since this ansible resource only from latest package